serialization.rst

```diff
@@ -183,14 +187,14 @@ possible mapping values are:
     is thrown with a message indicating that the class does not implement the
     expected interface.
```

Comment:
Shouldn't this be changed to say that if the class name encoded in ``__pclass`` does not implement ``MongoDB\BSON\Unserializable``, it should become an ``stdClass`` as well? Seems more consistent.

I also think that the two negative paragraphs (this one, and the one that starts with "If the named class does not exist" should be after the positive paragraph (the one that starts with "If the named class implements").